### DISCUSSION TITLE
Allow zooming into Note in Preview mode

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#759] [Feature Requests]
Feature request from a user migrating from Joplin.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by benji88i** [comment]
Hi my name is Benji and I love this project a lot. Happy to be apart of any support or testing of beta versions. 

I would like to vote on being able to zoom into the notes in read mode. This is because the text is too small to read on a Google pixel 4 which I believe is a common size screen.

**Comment by vHanda** [comment]
> Hi my name is Benji and I love this project a lot. Happy to be apart of any support or testing of beta versions.
> 
> I would like to vote on being able to zoom into the notes in read mode. This is because the text is too small to read on a Google pixel 4 which I believe is a common size screen.

Hey Benji. Nice to see you here. Please click on the '👍' button on the top. That way I can sort the issues based on which has the maximum number of '👍's.
